docs/machine-learning/automate-training-with-model-builder.md

```diff
@@ -13,11 +13,23 @@ Model Builder uses automated machine learning (AutoML) to explore different mach
 
 You don't need machine learning expertise to use Model Builder. All you need is some data, and a problem to solve. Model Builder generates the code to add the model to your .NET application.
 
-![Model Builder Visual Studio extension user interface animation](media/ml-dotnet-model-builder.gif)
+![Model Builder Scenarios](./media/model-builder-scenarios.png#lightbox)
 
 > [!NOTE]
 > Model Builder is currently in Preview.
 
+## Creating a Model Builder Project
+
+When you first start up Model Builder it will ask for you to name the project. This will create an `mbconfig` configuration file inside of the project.
+
+The `mbconfig` file keeps track of everything you do in Model Builder to allow you to reopen the session.
+
+After training, two files are generated under the *.mbconfig file:
+
```

Comment:
Add a note.

When you create your `mbconfig` file, you're prompted for a name. This name is applied to the consumption, training, and model files. In this case, the name used is *Model*.